submarine-server/server-core/src/test/java/org/apache/submarine/server/rest/ExperimentRestApiTest.java

```diff
@@ -224,7 +222,6 @@ private void verifyResult(Experiment experiment, String uid) {
     assertEquals(experimentFinishedTime, experiment.getFinishedTime());
     assertEquals(metaName, experiment.getSpec().getMeta().getName());
     assertEquals(metaFramework, experiment.getSpec().getMeta().getFramework());
-    assertEquals(metaNamespace, experiment.getSpec().getMeta().getNamespace());
```

Comment:
I think we should assert that the experiment namespace is default. And we should add a new test to make sure the namespace is what we expect if we update `ENV_NAMESPACE`